nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/extension/ConverterDescriptor.java

```diff
@@ -65,6 +65,9 @@
     @XNode("@type")
     protected String converterType = CUSTOM_CONVERTER_TYPE;
 
+    @XNode("@bypassIfSameMimeType")
+    protected boolean bypassIfSameMimeType = false;
```

Comment:
This new field should be handled in the `#merge` method to be able to override an existing converter and setting `bypassIfSameMimeType` to `false`.
You probably want to make it a `Boolean` instead to test if it was set or not in the contribution.

Add a unit test validating that we can override the `bypassIfSameMimeType` for a given converter.

Reply:
Also remove `= false` which is already the default for fields.